dotcom-rendering/src/components/Picture.tsx

```diff
@@ -12,6 +13,34 @@ import type { Loading } from './CardPicture';
 
 export type Orientation = 'portrait' | 'landscape';
 
+const timelineBulletStyles = css`
+	position: relative;
+	::before {
+		content: '';
+		position: absolute;
+		display: block;
+		width: 12px;
+		height: 12px;
+		border: 1px solid ${palette('--timeline-event-border')};
+		border-radius: 1000px;
+		background-color: ${palette('--timeline-bullet')};
+		top: -6px;
+		left: -7px;
```

Comment:
I think that if you technically want to align to the middle of a 1px width line you should nudge by a half pixel:

```suggestion
		left: -6.5px;
```

[See CodePen example](https://codepen.io/mxdvl/pen/poBqEqm?editors=1100)